src/Congo.WebApi/Startup.cs

```diff
@@ -24,8 +28,27 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddAppServices(Configuration);
-
             services.AddControllers();
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy(_CORS_DEV, policy =>
+                            policy.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader());
+
+                options.AddPolicy(_CORS_PROD, policy =>
+                {
+                    //TODO: Multiple Origins from env vars
```

Comment:
All comments should have a space between `//` and the actual text.
```suggestion
                    // TODO: Multiple Origins from env vars
```